OneDriveSDK/OneDriveCoreSDK/model/ODQuota.m

```diff
@@ -41,33 +41,33 @@ @interface ODQuota()
 @implementation ODQuota	
 
 
-- (NSInteger)deleted
+- (long long)deleted
 {
     
     if (self.dictionary[@"deleted"]){
-        return [self.dictionary[@"deleted"] intValue];
+        return [self.dictionary[@"deleted"] longLongValue];

```

Comment:
printf("size of long long: %lu, long: %lu , int: %lu \n\n",sizeof(long long),sizeof(long),sizeof(int));
size of long long: 8, long: 4 , int: 4 

Well, 4 bytes can only store value no large than 4G. 
So long long is  better.